task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -258,6 +258,16 @@ spec:
           mkdir -p "$YUM_REPOS_D_FETCHED"
           cp --no-clobber "$prefetched_repo_for_my_arch" "$YUM_REPOS_D_FETCHED"
         fi
+
+        # quotes within variable are not handled very well
+        # by ssh command and it impacts the buildah-remote task (which is based on this task)
+        # we only need to know if prefetch was used, not the actual
+        # input. Therefore, we simplify the variable.
+        #
+        if [ -n "${PREFETCH_INPUT}" ] ; then
+          PREFETCH_INPUT="prefetched"
+        fi
```

Comment:
What specifically is the problem? I don't see how this would help. Here we're manipulating the variable in the script executed in the podman container on the VM. If SSH mangles the variable value, it's already been mangled before this point.

Reply:
If there really is a problem, it might be better to stop using PREFETCH_INPUT and instead rely on checking `if [ -d "$(workspaces.source.path)/cachi2" ]`

Reply:
This is what I choose to do in the end. please review